imblearn/over_sampling/_smote.py

```diff
@@ -1306,9 +1267,9 @@ def _fit_resample(self, X, y):
                 )
 
                 self.nn_k_.fit(X_cluster_class)
-                nns = self.nn_k_.kneighbors(
-                    X_cluster_class, return_distance=False
-                )[:, 1:]
+                nns = self.nn_k_.kneighbors(X_cluster_class, return_distance=False)[
+                    :, 1:
+                ]
```

Comment:
Is this spilling over the max line length?

This looks like a case where `black` misses eliminating newlines.